test/e2e/framework/timeouts.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import "time"
+
+// TimeoutContext contains timeout settings for several actions.
+type TimeoutContext struct {
```

Comment:
There is lot of timeouts in this struct that are never used in this PR. Does it make sense to pre-allocate them for subsequent, non-storage PRs?

Reply:
My original idea was to:

1. Move all timeouts to this struct
2. Deprecate the original values used here: https://github.com/kubernetes/kubernetes/blob/f975a2d485ca7ce559d437ed6a1b07111bf1404d/test/e2e/framework/util.go#L86
3. Use the values from this struct _in all e2e tests_, not only storage tests.

However, when I got to step 3 above I noticed that this PR would become excessively big, so I started with storage tests only and hoped that eventually the other tests would use these values instead (since the values in utils.go are deprecated).

Does this make sense or should I just add to this struct the timeouts I'm going to use in storage tests?

The reason I wanted to do this is because currently timeout values are scattered around framework package: utils.go, pv/pv.go etc. So I'd like this file to be the source of truth for all values.

Reply:
Keeping the timeouts in a single place (timeouts.go) makes the code readable. I think we can start with the timeouts we are using in storage tests.

Reply:
Sounds good. Removed everything that was not used.